app/models/ott/CategoryAssessment.scala

```diff
@@ -34,6 +36,13 @@ final case class CategoryAssessment(
   def getExemptionListItems: Seq[String] = exemptions.map { exemption =>
     exemption.code + " - " + exemption.description
   }
+
+  def isNiphlsAnswer: Boolean =
+    category == 1 && exemptions.exists(ex =>
```

Comment:
Can we replace 1s and 2s with the constant CATEGORY1 etc.

Reply:
Done